doc/response-tagging.rst

```diff
@@ -0,0 +1,76 @@
+Response Tagging
+================
+
+This library provides a helper to handle tagging of responses ``ResponseTagger``
+to simplify use of tags. The ``ResponseTagger`` helps you keep track tags for a response, which can be
```

Comment:
i think the first line can simply be removed, it does not say anything more than the second line.